packages/synapse-interface/utils/providers/SynapseProvider.tsx

```diff
@@ -2,23 +2,39 @@
 import { SynapseSDK } from '@synapsecns/sdk-router'
 import { Provider } from '@ethersproject/abstract-provider'
 import { createContext, useContext, memo, useMemo } from 'react'
-import { StaticJsonRpcProvider } from '@ethersproject/providers'
+import {
+  StaticJsonRpcProvider,
+  FallbackProvider,
+  FallbackProviderConfig,
+} from '@ethersproject/providers'
 import { Provider as EthersProvider } from '@ethersproject/abstract-provider'
 
 export const SynapseContext = createContext(null)
 
 export const SynapseProvider = memo(
   ({ children, chains }: { children: React.ReactNode; chains: any[] }) => {
     const synapseProviders = useMemo(() => {
-      return chains.map(
-        (chain) => new StaticJsonRpcProvider(chain.configRpc, chain.id)
-      )
+      return chains.map((chain) => {
+        const providerUrls = [chain?.configRpc, chain?.fallbackRpc]
```

Comment:
Could `chain` really be undefined here?

Reply:
Working through this to see if we can define out the `chain` type passed in here 

For context, we are adding a few props to the Wagmi `Chain` type when calling `configureChains()` [here](https://github.com/synapsecns/sanguine/blob/fe/fallback-provider/packages/synapse-interface/pages/_app.tsx/#L81-L93)

Reply:
Looks like we currently have a version mismatch which is causing for types to not pass properly between `wagmi` `viem` and `@rainbow-me/rainbowkit` packages

Can we leave `chain` as `any` type for now and address version upgrades + typing `chain` in a later PR?

Reply:
Sorry, misinterpreted your comment! I see what you're saying. I intentionally left it as potentially undefined since the `chains` prop is currently typed as `any[]`, wanted to prevent any potential app crashes due to `chain` becoming undefined due to some change in the future from external package upgrades (viem / wagmi)

Going to leave as is for now and update when we have proper typing for props passed down into `SynapseProvider`